src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -127,8 +139,10 @@ export default {
 					// ignore
 				}
 				let subline = vobject.calendar.displayName
+				let color = null
```

Comment:
Not sure if needed, `VTODO` and `VJOURNAL` can have colors as well.

Reply:
Done. Now the color of the first component is tried to be used regardless of its type.